src/core/defs.jl

```diff
@@ -132,17 +132,29 @@ description(def::DatumDef) = def.description
 
 unit(def::DatumDef) = def.unit
 
+# LFR-TBD:  Since not all models have a step_size, we have removed this function
+# and replaced it with a complementary one, yeras_array
+
 # step_size(md::ModelDef) = step_size(indexvalues(md))
 
-function step_size(md::ModelDef)
-    # N.B. assumes that all timesteps of the model are the same length
-    keys::Vector{Int} = dim_keys(md, :time) # keys are, e.g., the years the model runs
-    return length(keys) > 1 ? keys[2] - keys[1] : 1
+# function step_size(md::ModelDef)
+#     # N.B. assumes that all timesteps of the model are the same length
+#     keys::Vector{Int} = dim_keys(md, :time) # keys are, e.g., the years the model runs
+#     return length(keys) > 1 ? keys[2] - keys[1] : 1
+# end
+
+# function step_size(values::Vector{Int})
+#     # N.B. assumes that all timesteps of the model are the same length
+#      return length(values) > 1 ? values[2] - values[1] : 1
+# end
+
+function years_array(md::ModelDef)
+    keys::Vector{Int} = dim_keys(md, :time)
+    return keys
 end
 
-function step_size(values::Vector{Int})
-    # N.B. assumes that all timesteps of the model are the same length
-     return length(values) > 1 ? values[2] - values[1] : 1
+function years_array(values::Vector{Int})
```

Comment:
What's the purpose if `years_array(values) = values` ?

Reply:
Good question, I'll take a look.  I was mainly just copying the `step_size` functions so that I can replace uses of `step_size` with `years_array`when needed, but this is probably superfluous.